guides/release/components/built-in-components.md

```diff
@@ -1,43 +1,42 @@
-Out of the box, Ember provides 2 components for building a form:
+Ember fournit deux composants prêts à l'emploi pour construire des formulaires:
 
 * [`<Input>`](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/Input?anchor=Input)
 * [`<Textarea>`](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/Textarea?anchor=Textarea)
 
-These components are similar in HTML markup to the native `<input>` or `<textarea>` elements. In contrast to the native elements, `<Input>` and `<Textarea>` automatically update the state of their bound values.
-
+Le markup HTML de ces composants est similaire à celui des éléments natifs `<input>` et `<textarea>`. À la différence des éléments natifs, `<Input>` et `<Textarea>` mettent automatiquement à jour l'état des valeurs qui leur sont liées.
 
 ## `<Input>`
 
-We mentioned that the built-in components are similar in HTML markup to their native counterparts. What does this mean?
+Nous avons dit plus haut que les composants _built-in_ (c'est-à-dire "construits à l'intérieur du framework") ont un markup HTML similaire à celui de leur équivalent natif. Que'est-ce que ça signifie ?
 
-Consider the following example in a template file.
+Considérons l'exemple suivant, le template d'un fichier.
 
 ```handlebars
-<label for="user-question">Ask a question about Ember:</label>
+<label for="user-question">Posez une question sur Ember :</label>
 <Input
   id="user-question"
   @type="text"
-  @value="How do text fields work?"
+  @value="Comment fonctionnent les champs textes ?"
```

Comment:
Champs de textes?

Reply:
It's ok to write "champ texte", but I shouldn't have put a "s" at the end of "texte", because it's not an adjective, it's a shortcut to "de type texte"  

```suggestion
  @value="Comment fonctionnent les champs texte ?"
```